src/renderer/components/chart/pie-chart.tsx

```diff
@@ -16,6 +16,17 @@ import { ThemeStore } from "../../theme.store";
 export interface PieChartProps extends ChartProps {
 }
 
+export interface PieChartData extends ChartJS.ChartData {
+  datasets?: PieChartDataSets[];
+}
+
+export type DatasetTooltipLabel = (percent: string) => string | string;
+
+interface PieChartDataSets extends ChartJS.ChartDataSets {
+  id?: string;
+  tooltipLabels?: DatasetTooltipLabel[];
```

Comment:
I'm wondering why `id` wasn't added directly to `ChartJS.ChartDataSets` interface directly if those are supported in some way? Also maybe rename `tooltipLabels`  to `formatTooltips`?

Reply:
`id`s isn't supported by chartjs natively, we're using them internally in `src/renderer/components/chart/chart.tsx`

Reply:
`tooltipLabels` already used in https://github.com/lensapp/oci-image-lens-extension/pull/303/files